binaries/cli/src/command/build/local.rs

```diff
@@ -98,24 +114,53 @@ impl BuildLogger for LocalBuildLogger {
         level: impl Into<LogLevelOrStdout> + Send,
         message: impl Into<String> + Send,
     ) {
-        let level = match level.into() {
-            LogLevelOrStdout::LogLevel(level) => match level {
-                log::Level::Error => "ERROR ".red(),
-                log::Level::Warn => "WARN  ".yellow(),
-                log::Level::Info => "INFO  ".green(),
-                log::Level::Debug => "DEBUG ".bright_blue(),
-                log::Level::Trace => "TRACE ".dimmed(),
-            },
-            LogLevelOrStdout::Stdout => "stdout".italic().dimmed(),
-        };
-        let node = self.node_id.to_string().bold().bright_black();
-        let message: String = message.into();
-        println!("{node}: {level}   {message}");
+        let message_str: String = message.into();
+
+        // Update progress bar message if available
+        if let Some(pb) = &self.progress_bar {
+            let level_indicator = match level.into() {
+                LogLevelOrStdout::LogLevel(level) => match level {
+                    log::Level::Error => "ERROR",
+                    log::Level::Warn => "WARN",
+                    log::Level::Info => "",
+                    log::Level::Debug => "DEBUG",
+                    log::Level::Trace => "TRACE",
+                },
+                LogLevelOrStdout::Stdout => "",
+            };
+
+            if !level_indicator.is_empty() {
+                pb.set_message(format!(
+                    "{}: {} - {}",
+                    self.node_id, level_indicator, message_str
+                ));
+            } else {
+                pb.set_message(format!("{}: {}", self.node_id, message_str));
+            }
+        } else {
+            // Fallback to println if no progress bar
```

Comment:
I'm not sure if we want to "hide" the log messages by default. It might be good to still have the output in the terminal so that you can scroll up later to look for warnings or errors.